microsoft/surface/repos.nix

```diff
@@ -1,8 +1,26 @@
-{ lib, pkgs, fetchgit }:
+{ lib, pkgs, fetchgit, fetchurl }:
 {
   linux-surface = fetchgit {
```

Comment:
also switch to fetchFromGithub here since it is faster especially for large repositories.

Reply:
OK, I wasn't aware of that improvement.